File: NPB-CPP-cadna/cadna_c-3.1.11/examplesFortran/ex5.f90
      program ex5
      implicit double precision (a-h,o-z)
      parameter (eps = 1.0d-12, nmax = 100)
      !
      print *,'Computation of a root of a polynomial by Newton method'
      print *,'without CADNA:'
      y = 0.5d0
      do i = 1,nmax
        x = y
        y = x - (1.47d0*x**3 + 1.19d0*x**2 - 1.83d0*x + 0.45d0)&
     &         /(4.41d0*x**2 + 2.38d0*x - 1.83d0)
        if (abs(x-y).le.eps) goto 1
      enddo
!
 1    continue
!
      write(*,*)'x(',i-1,') = ',x
      write(*,*)'x(',i,') = ',y
!
      end

File: NPB-CPP-cadna/cadna_c-3.1.11/examplesC/ex5.c
#include <math.h>
#include <stdio.h>

int main()
{
  int i, nmax=100;
  double y, x, eps=1.e-12; 
  printf("-------------------------------------------------------------\n");
  printf("|  Computation of a root of a polynomial by Newton's method |\n");
  printf("|  without CADNA                                            |\n");
  printf("-------------------------------------------------------------\n");
  
  y = 0.5;
  for(i = 1;i<=nmax;i++){
    x = y;
    y = x-(1.47*x*x*x+1.19*x*x-1.83*x+0.45) / (4.41*x*x+2.38*x-1.83); 
    if (fabs(x-y)<=eps) break;
  }
  printf("x(%3d) = %+.15e\n",i-1,x);
  printf("x(%3d) = %+.15e\n",i,y);
  return 0;
}
